src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+  box-sizing: border-box;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
```

Comment:
Using `!important` - says that you are doing something wrong, it can be used only in extreme cases.

Reply:
I've just thought that h1 is necessary in html document. And so i just hide it with bootstrap's visualy hiden class. I know that its unnecessary right here but is this wrong? @roman-mirzoian

Reply:
@kaavka https://w3schoolsua.github.io/css/css_important.html#gsc.tab=0